### PR TITLE
[ Sample2] after last pull - request adding new file

### PR DESCRIPTION
**Pull request by zaki2030**
please, accept

### CODE REVIEW

File: test3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a new change from zaki2030
```
Comment (zaki2027) [inline test3.txt]: another great change, edit it

**Review by zaki2027 — COMMENTED** [review]
not enough

**Review by zaki2027 — COMMENTED** [review]
again

**Comment by zaki2030** [comment]
> 

what do you think ?

**Comment by zaki2027** [comment]
great job

**Review by zaki2030 — COMMENTED** [review]
that is too great :)

File: test3.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+editing it
+
+
+hope this is enough.
+
+final
```
Comment (zaki2030) [inline test3.txt:8]: thanks

**Pull request merged**